docs/advanced/RendererConfiguration.md

```diff
@@ -12,6 +12,7 @@ We might introduce more configuration options with future releases, so be sure t
 | keyframePrefixes | *(Array?)* |`['-webkit-',`<br>`'-moz-']` | A list of which additional `@keyframes` prefixes are rendered |
 | enhancers  | *(Array?)* |  |  A list of [enhancers](../advanced/Enhancers.md) to enhance the renderer
 | mediaQueryOrder | *(Array?)* |  | An explicit order in which `@media` queries are rendered |
+| rendererId | *(string?)* |  | An optional unique identifier that will prefix your animation names and will be used in `<style>` nodes. |
```

Comment:
I feel this isn't clear enough.

Why would someone use this?  What is the use case? Why might someone want to perfix their animation names and what's the connection between animation names and the attribute on the style nodes?

Without knowing in advance, I couldn't have figured it out from this description.

Also, I think it's be clearer if it were `will be added as the value of a data-fela-id attribute on <style> nodes` rather than `used in <style> nodes`

Reply:
This is much cleaner 👍 Updated

Reply:
Thanks. It's clear now _what_ it does, but still not really clear what one would use that for.

I think it'd be good to describe the use case

Reply:
> An example use case: the recommended way is to have only one renderer, but there are some specific cases when you will need to have multiple renderers. This option allows you avoid conflicts in generated CSS rules via prefixing animation names and separated `<style>` tags.

Going to add this? Do you have any suggestions?

Reply:
Yeah, that's great. Thanks

Reply:
Pushed 👍